tests/net/context/src/main.c

```diff
@@ -786,58 +659,48 @@ static bool net_ctx_sendto_v4_wrong_src(void)
 	return true;
 }
 
-static bool net_ctx_recv_v4_fail(void)
+static void net_ctx_recv_v4_fail(void)
 {
 	net_ctx_sendto_v4_wrong_src();
 
 	if (!k_sem_take(&wait_data, WAIT_TIME)) {
-		TC_ERROR("Semaphore triggered but should not\n");
-		return false;
+		zassert_true(0, "Semaphore triggered but should not");
```

Comment:
can you check for the return directly in the assert?

Reply:
Done